changelogs/fragments/1735-imc-sessions.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - imc_rest - explicitly logging out instead of registering the call in ```atexit``` (https://github.com/ansible-collections/community.general/issues/1735).
```

Comment:
```suggestion
  - imc_rest - explicitly logging out instead of registering the call in ``atexit`` (https://github.com/ansible-collections/community.general/issues/1735).
```